### ISSUE TITLE
Shortcuts Manager, shortcuts do not work

### ISSUE THREAD

**Issue by crescentduchess** [Ashung/Automate-Sketch#234]
Tried both methods as described in Shortcuts Manager but pressing the keys has no effect. No duplicate shortcuts. Tried restarting and going through system shortcuts. No effect, regardless of plugin menu.

**Comment by Ashung** [comment]
You can click the gray text in the list to copy the string, then go to System Preferences - Keyboard - Shortcuts to add a shortcut.

**Comment by kirchenn** [comment]
Hi Ashung! The solution with system preferences still doesn't work though, it shows that such a shortcut was applied, but pressing it doesn't do anything.

**Comment by richardsison** [comment]
FWIW this might be macOS issue or a Sketch issue. From what @kirchenn was saying, it sounds as though the menu item flashes but doesn't actually do anything. I had a similar issue with the mail app [Postbox](https://www.postbox-inc.com/) where navigating through the menu items activated bullet lists, but setting a shortcut to it didn't.

I've set several shortcuts to Automate Sketch via System Preferences and they all work so unless you're trying to map an action that can't be mapped to a shortcut key (for whatever reason), maybe you just have a keyboard shortcut conflict with macOS or another app.

Have you tried setting the shortcut to something really obscure that another app likely wouldn't conflict with? For example, `⌃` + `⌥` + `⌘` + `0` isn't likely to be taken by anything.

But for an example of a shortcut I've mapped and use multiple times a day, try mapping
`Plugins->Automate->Layer->Paste and Replace` to `⌥` + `⌘` + `R` (I know for a fact that this works).

Hope that helps!
